app/src/main/java/org/simple/clinic/drugs/EditMedicinesEvent.kt

```diff
@@ -24,6 +24,8 @@ object PrescribedDrugsDoneClicked : EditMedicinesEvent() {
   override val analyticsName = "Drugs:Protocol:Save Clicked"
```

Comment:
Can we rename this commit to `Refill medicines when refill button is clicked`, please?